compiler/stable_mir/src/ty.rs

```diff
@@ -460,7 +460,7 @@ pub enum RigidTy {
     FnDef(FnDef, GenericArgs),
     FnPtr(PolyFnSig),
     Closure(ClosureDef, GenericArgs),
-    Coroutine(CoroutineDef, GenericArgs, Movability),
```

Comment:
I was wondering if we could leave this here for now and make it always the same value. We could add a comment documenting that for now?

BTW, we need to figure out the best mechanism to make breaking changes to StableMIR, but this one I believe we can delay this one for now.

Reply:
What do you mean by making it always the same value?

I'd rather expose a `fn movability(CoroutineDef) -> Movability` fn in the long term, but I can "pre-populate" it here I guess for the mean time.

Reply:
Yes, that would work. I misunderstood the reason why you were removing it.

Reply:
Which would you like for me to do first? Make `smir::TyKind::Coroutine` keep the `Movability` argument even though it's redudant, or add a `fn movability(CoroutineDef) -> Movability` to the tables?

Reply:
I've just restored movability to SMIR, and added a `FIXME(stable_mir)`.